docs/setup/data/network-config.txt

```diff
@@ -21,11 +21,11 @@ ethernets:
 #    dhcp4: true
 #    optional: true
 #    access-points:
-#      myhomewifi:
+#      "myhomewifi":
```

Comment:
I ran into this just the other day! IIRC the Pi Imager doesn't produce the right output or else the Canonical docs are wrong (can't remember which was wrong, but something was)

Reply:
I think it is that the docs are wrong? It was a major issue this summer that Sebastian and I spent a long time trying to figure out what we were doing wrong. I finally remembered to include it in this PR